group/templates/group/groupDetail.html

```diff
@@ -0,0 +1,519 @@
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<!DOCTYPE html>
+<head>
+  <meta charset="utf-8">
+  <title>Group Detail - NTHU Online Judge</title>
+  <!-- Bootstrap&Somethings styles -->
+  <link rel="stylesheet" href="http://bootswatch.com/simplex/bootstrap.css" media="screen">
+  <link href="http://bootswatch.com/paper/bootstrap.css" rel="stylesheet">
+  <link rel="stylesheet" href="http://bootswatch.com/assets/css/bootswatch.min.css">
+
+  <script src="http://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>
+  <script src="http://libs.baidu.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
+  <!-- Magnific Popup core JS & CSS -->
+  <link rel="stylesheet" href="/css/magnific-popup.css">
+  <link rel="stylesheet" href="/css/mfp-fade.css">
+  <script src="/js/jquery.magnific-popup.js"></script>
+  <script src="/js/popup-setting.js"></script>
+</head>
+
+{% extends "index/base.html" %}
+{% load static %} 
+{% block body_block %}
+
+<script language="javascript" type="text/javascript" 
+        src="//cdnjs.cloudflare.com/ajax/libs/flot/0.8.2/jquery.flot.min.js"></script>
+<script language="javascript" type="text/javascript" 
+        src="//cdnjs.cloudflare.com/ajax/libs/flot/0.8.2/jquery.flot.pie.min.js"></script>
+
+<div class="container-fluid">
+  <div class="row-fluid">
+    <a href="/group/list" class="btn btn-sm btn-default" data-toggle="modal" style="float:left">
+      return
+    </a>
+    <br>
+    <br> 
+    {% block Group_intro_block %}
+    <div class="span12">
+      <div class="hero-unit">
+        <div class="hero-unit">
+          <span style="width:50%; float:left">
+            <p>Group name</p>
+            <h2>{{ group_name }}</h2>
+          </span>
+          <span style="width:50%; float:right">
+              <p>Owner/TA</p>
+              <button class="btn btn-default btn-sm" data-toggle="modal" data-target="#Setowner" 
+                      style="float:right">+</button>
+              <h2>
+                {% for ta in ta_name %}
+                  {{ ta.name }}
+                {% endfor %} 
+              </h2>
+              </span>
+          <br>
+          <br>
+          <br>
+        </div>
+        <hr>
+        <div class="Description">
+          <p style="font-size:16px;color:gray">
+            {{ group_description }}
+            <a href="" class="btn btn-sm btn-default" data-toggle="modal" data-target="#myModal"
```

Comment:
I hope the naming of data-target here is more meaningful.
such as "groupDescriptionModal" ...
You can discuss the naming rules about such kind of variables.